src/handlers/query-wallet.ts

```diff
@@ -21,24 +22,29 @@ export async function registerWallet(context: Context, body: string) {
   const ensName = extractEnsName(body.replace("/wallet", "").trim());
 
   if (!address && ensName) {
-    context.logger.debug("Trying to resolve address from ENS name", { ensName });
+    logger.debug("Trying to resolve address from ENS name", { ensName });
     address = await resolveAddress(ensName);
     if (!address) {
       throw new Error(`Resolving address from ENS name failed: ${ensName}`);
     }
-    context.logger.debug("Resolved address from ENS name", { ensName, address });
+    logger.debug("Resolved address from ENS name", { ensName, address });
   }
 
   if (!address) {
-    return context.logger.info("Skipping to register a wallet address because both address/ens doesn't exist");
+    const logMessage = logger.info("Skipping to register a wallet address because both address/ens doesn't exist");
+    await addCommentToIssue(context, logMessage.logMessage.diff);
```

Comment:
```suggestion
    await addCommentToIssue(context, logger.info("Skipping to register a wallet address because both address/ens doesn't exist").logMessage.diff);
```

you can directly inline it to male it simpler.